x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -430,6 +430,7 @@ export class TaskRunner<
         `rule execution failure: ${ruleLabel}`,
         err.message
       );
+      this.logger.error(err);
```

Comment:
Logging when rule type executors fail immediately, otherwise the alerting task runner rethrows the error later and the stack trace from within the executor is lost.